src/Pos/Client/Txp/History.hs

```diff
@@ -237,15 +236,16 @@ getBlockHistoryDefault
     :: forall ssc ctx m. TxHistoryEnv' ssc ctx m
     => [Address] -> m (DList TxHistoryEntry)
 getBlockHistoryDefault addrs = do
-    bot <- GS.getBot
-    sd <- GS.getSlottingData
-    systemStart <- getSystemStart
+
+    systemStart <- getSystemStartM
+    bot         <- GS.getBot
+    sd          <- GS.getSlottingData
```

Comment:
do we still store `SlottingData` in the database?
I thought we will get rid of that datatype.